mevolib/inference/__init__.py

```diff
@@ -174,3 +175,43 @@ def get_phylogeny(
         # Return the resultant phylogeny as a Bio.Phylo.BaseTree object and its
         # log-likelihood score
         return phylogeny, score
+
+def main():
+    """Default call for Inference module."""
+    parser = argparse.ArgumentParser(
+        description="Infers the phylogeny of a given set of aligned genes using the given inference"
+        + " tool and arguments"
+    )
+    parser.add_argument(
```

Comment:
This will add the default value automatically in your help message:
```suggestion
    parser.formatter_class = argparse.ArgumentDefaultsHelpFormatter
    parser.add_argument(
```